src/Controls/src/Core/Handlers/Shell/Windows/ShellView.cs

```diff
@@ -193,6 +193,12 @@ public override bool Equals(object obj) =>
 
 		internal void SwitchShellItem(ShellItem newItem, bool animate = true)
 		{
+			if (newItem is null)
```

Comment:
What's the behavior on other platforms? Would be nice to align the behavior and throw the same exception with the same message if possible.

Reply:
@jsuarezruiz, On Android, the empty Shell throws a similar exception.